JetMETCorrections/Modules/plugins/METCorrectorDBReader.cc

```diff
@@ -0,0 +1,143 @@
+// -*- C++ -*-
+//
+// Package:    METCorrectorDBReader
+// Class:      
+// 
+/**\class METCorrectorDBReader
+
+ Description: Reads out *.db format for MET corrections 
+
+ Implementation:
+     <Notes on implementation>
+*/
+//
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "CondFormats/JetMETObjects/interface/METCorrectorParameters.h"
+#include "JetMETCorrections/Objects/interface/METCorrectionsRecord.h"
+#include "JetMETCorrections/Objects/interface/JetCorrectionsRecord.h"
+
+
+//
+// class declaration
+//
+namespace{
+
+class METCorrectorDBReader : public edm::one::EDAnalyzer<> {
+public:
+  explicit METCorrectorDBReader(const edm::ParameterSet&);
+  ~METCorrectorDBReader();
+  
+  
+private:
+  virtual void beginJob() override ;
+  virtual void analyze(const edm::Event&, const edm::EventSetup&) override;
+  virtual void endJob() override ;
+ 
+  std::string mPayloadName,mGlobalTag;
+  bool mCreateTextFile,mPrintScreen;
+};
+
+}
+
+
+METCorrectorDBReader::METCorrectorDBReader(const edm::ParameterSet& iConfig)
+{
+  mPayloadName    = iConfig.getUntrackedParameter<std::string>("payloadName");
+  mGlobalTag      = iConfig.getUntrackedParameter<std::string>("globalTag");  
+  mPrintScreen    = iConfig.getUntrackedParameter<bool>("printScreen");
+  mCreateTextFile = iConfig.getUntrackedParameter<bool>("createTextFile");
+}
+
+
+METCorrectorDBReader::~METCorrectorDBReader()
+{
+ 
+}
+
+void METCorrectorDBReader::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
+{
+  edm::ESHandle<METCorrectorParametersCollection> METCorParamsColl;
+  std::cout <<"Inspecting MET payload with label: "<< mPayloadName <<std::endl;
```

Comment:
`cout` is not allowed in mainstream code. Please change to LogDebug/LogInfo as appropriate.